processor/resourcedetectionprocessor/internal/aws/ecs/metadata.yaml

```diff
@@ -35,6 +35,10 @@ resource_attributes:
     description: The aws.ecs.task.family
     type: string
     enabled: true
+  aws.ecs.task.id:
+    description: The aws.ecs.task.id
```

Comment:
Please add a description of this. For the other fields, the description is on the semantic conventions repository, but this one has not description anywhere. You may also want to update the processor's README

Reply:
Hey @mx-psi,
Thanks for spotting that. I've updated README to mention new attribute, and I've created a new PR: https://github.com/open-telemetry/semantic-conventions/pull/597